### ISSUE TITLE
Continue parsing even with illegal characters

### ISSUE THREAD

**Issue by jakeri** [FasterXML/aalto-xml#15]
I know this is not correct but I am trying to parse some sources that have illegal characters in the xml feed.
Would it be possible to let the parser continue parse without exception and warn instead?

``` text
com.fasterxml.aalto.WFCException: Illegal XML character ((CTRL-CHAR, code 11))
 at [row,col {unknown-source}]: [29273,485]
    at com.fasterxml.aalto.in.XmlScanner.reportInputProblem(XmlScanner.java:1335) ~[aalto-xml-0.9.8.jar:na]
    at com.fasterxml.aalto.in.XmlScanner.throwInvalidXmlChar(XmlScanner.java:1525) ~[aalto-xml-0.9.8.jar:na]
    at com.fasterxml.aalto.async.AsyncUtfScanner.skipCharacters(AsyncUtfScanner.java:755) ~[aalto-xml-0.9.8.jar:na]
    at com.fasterxml.aalto.async.AsyncByteScanner.nextFromTree(AsyncByteScanner.java:572) ~[aalto-xml-0.9.8.jar:na]
    at com.fasterxml.aalto.stax.StreamReaderImpl.next(StreamReaderImpl.java:720) ~[aalto-xml-0.9.8.jar:na]
    ...
```

and

``` text
com.fasterxml.aalto.WFCException: Illegal XML character ((CTRL-CHAR, code 22))
 at [row,col {unknown-source}]: [8982,3]
    at com.fasterxml.aalto.in.XmlScanner.reportInputProblem(XmlScanner.java:1335)
    at com.fasterxml.aalto.in.XmlScanner.throwInvalidXmlChar(XmlScanner.java:1525)
    at com.fasterxml.aalto.async.AsyncUtfScanner.skipCharacters(AsyncUtfScanner.java:755)
    at com.fasterxml.aalto.async.AsyncByteScanner.nextFromTree(AsyncByteScanner.java:572)
    at com.fasterxml.aalto.stax.StreamReaderImpl.next(StreamReaderImpl.java:720)
    ...
```

**Comment by philippe-tr** [comment]
It does not seem as though this pull request made it into a release. The last published release is from 2015, version 1.0.0.  Is this scheduled to make it into an upcoming release?

**Comment by cowtowncoder** [comment]
@philippe-tr I think PR in question was #16 and was included in 0.9.9.
